include/crow/app.h

```diff
@@ -68,7 +68,7 @@ namespace crow
         }
 
         /// Process the request and generate a response for it
-        void handle(const request& req, response& res)
```

Comment:
Remove const qualifiers up to rule->handle() to allow passing mutable requests into local middleware